src/quote/swap/adapters/zeroex.ts

```diff
@@ -118,7 +123,22 @@ export class ZeroExSwapQuoteProvider implements SwapQuoteProvider {
     try {
       const response = await axios.get(url, config)
       const res: ZeroExApiSwapResponse = response.data
-      return res
+      return {
+        chainId,
+        inputToken,
+        outputToken,
+        inputAmount: res.sellAmount,
+        outputAmount: res.buyAmount,
+        callData: res.data,
+        slippage: slippage ?? 0,
+        // TODO: add swap data
+        swapData: {
+          exchange: Exchange.UniV3,
+          path: ['', ''],
+          fees: [300],
+          pool: '0x0000000000000000000000000000000000000000',
+        },
```

Comment:
That#s why this is all just fake swap data for now

Reply:
was going to ask about it - thank you